uswds/uswds
# USWDS - Dependencies: Flush dependency cache and rebuild package-lock.json

# Summary

Cleaned npm cache and ran a clean install.

## Breaking change

This is not a breaking change.

## Related issue

Closes #5929

## Related pull requests

Originally discussed in #5782 

## Preview link

Storybook preview →

## Problem statement

Our cache should be cleaned periodically to ensure `package-lock` is up to date and sub-dependencies are updated.

## Solution

Clean cache and rebuild `package-lock.json` file.

- cleaned the npm cache via `npm cache clean -f`
- removing node_modules in project folder
- removing the package-lock.json file
- running npm install in now "clean" project folder

## Major changes

Sub-dependencies are updated.

## Testing and review

1. Run npm install and confirm there are no errors
1. Run npm start.
1. Update a SASS file and see it update in StorybookJS.
1. Running npx gulp sassTests or npm run test should not fail.
1. Run gulp tasks (like build) and ensure there aren't errors and things build correctly
1. Installing on site does not cause any installation or build errors

## Dependency updates

No direct dependency updates.
